### ISSUE TITLE
Object.dirname is deprecated.

### ISSUE THREAD

**Issue by Parsifal** [blakawk/gitlab-integration#3]
Argument to `path.dirname` must be a string
```
Object.dirname (/private/var/folders/3s/jy1ptlss4czb8ph2cr_lkb640000gn/T/AppTranslocation/C6C398B3-DC36-47B3-8E2F-1BC9436EF748/d/Atom.app/Contents/Resources/app/src/electron-shims.js:9:10)
File.getParent (<embedded>:112702:45)
GitlabIntegration.onPathChange (/Users/parsifal/.atom/packages/gitlab-integration/lib/gitlab-integration.coffee:70:47)
<unknown> (/Users/parsifal/.atom/packages/gitlab-integration/lib/gitlab-integration.coffee:157:19)
Function.module.exports.Emitter.simpleDispatch (<embedded>:63093:20)
Emitter.emit (<embedded>:63209:34)
```

**Comment by blakawk** [comment]
@Parsifal can you provide version information (at least atom core and gitlab-integration) ? Thanks.

**Comment by onza** [comment]
Same: Atom 1.18.0, gitlab-integration 0.2.3, macos 10.12.5

**Comment by blakawk** [comment]
@Parsifal @onza should be fixed in 0.2.4, please tell me if not !
